docs/modules/configuring/con-configuring-kafkaservice-cipher.adoc

```diff
@@ -0,0 +1,31 @@
+[id='con-configuring-kafkaservice-cipher-{context}']
+= Controlling the TLS ciphers used
+
+Some cipher suites, which are enabled by default for interoperability purposes, are no longer consider secure.
+
+IMPORTANT: It is considered good practice to disable insecure cipher suites. You can only do this is the proxied Kafka cluster you want to connect to supports the cipher suites that you want to use.
+
+The cipher suites that are enabled by default can depend on the JVM used in the proxy image and the TLS protocol version which is negotiated.
```

Comment:
```suggestion
The cipher suites enabled by default depend on the JVM used in the proxy image and the TLS protocol version that is negotiated.
```

Reply:
I would change the order and bring this together with the sentence on _some cipher suites_